CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 2024-10-22
+### Added
+- Updated default [`version`](https://github.com/overleaf/toolkit/blob/master/lib/config-seed/version) to `5.2.0`.
+- Drop support for Docker Compose v1. [How to switch to Compose V2](https://docs.docker.com/compose/releases/migrate/#how-do-i-switch-to-compose-v2).
+
```

Comment:
```suggestion
  Docker Compose v1 has reached its End Of Life in July 2023.

```
WDYT about adding this as a reminder why we are dropping support?

Reply:
Good idea.